utils/ssh.py

```diff
@@ -230,7 +233,9 @@ def status(self):
             contains dictionaries, one per line. You can refer inside the dictionary using the
             headers.
         """
-        data = self.run_command("service evmserverd status")
+        data = self.run_command({
+            version.LOWEST: "service evmserverd status",
+            "5.5": "cd /var/www/miq/vmdb; bin/rake evm:status"})
```

Comment:
Can we rather go with [`run_rake_command`](https://github.com/RedHatQE/cfme_tests/blob/master/utils/ssh.py#L180) here?